os/fs/driver/mtd/smart.c

```diff
@@ -5854,6 +5854,17 @@ static int smart_journal_recovery(FAR struct smart_struct_s *dev, journal_log_t
 
 #endif
 
+/****************************************************************************
+ * Name: get_smart_mtd_header_size
+ *
+ * Description: Return the size of the smart_sect_header_structure
+ *
+ ****************************************************************************/
+int get_smart_mtd_header_size()
```

Comment:
We usually name the function as MODULENAME_VERB_OBJECTIVES.
So I think `smart_get_smart_mtd_header_size` or `smart_get_mtd_header_size` is better.

Reply:
@amogh-samsung I also agree with this suggestion.